pkg/flowkit/config/json/account_test.go

```diff
@@ -362,7 +362,22 @@ func Test_TransformAccountToJSON(t *testing.T) {
 
 	assert.Equal(t, string(b), string(x))
 }
+func Test_TransformDefaultAccountToJSONAdvanced(t *testing.T) {
+	b := []byte(`{"emulator-account":{"address":"f8d6e0586b0a20c7","key":{"type":"hex","index":0,"signatureAlgorithm":"ECDSA_P256","hashAlgorithm":"SHA3_256","privateKey":"1272967fd2bd75234ae9037dd4694c1f00baad63a10c35172bf65fbb8ad74b47"}},"testnet-account":{"address":"3c1162386b0a245f","key":"2272967fd2bd75234ae9037dd4694c1f00baad63a10c35172bf65fbb8ad74b47"}}`)
+
+	var jsonAccounts jsonAccounts
+	err := json.Unmarshal(b, &jsonAccounts)
+	assert.NoError(t, err)
+	accounts, err := jsonAccounts.transformToConfig()
+	assert.NoError(t, err)
+
+	accounts[0].UseAdvancedSaveFormat = true
```

Comment:
I think this line is redundant now that the parser automatically identifies advanced accounts.

I would construct this test similarly to the one above:

```go
account := defaultAccount(t)
account.EnabledAdvancedSaveFormat()

// ... test that it results in advanced JSON
```

Reply:
you are right. Removing it.